Patch by Robert Scheck <robert@fedoraproject.org> for unicornscan >= 0.4.7, which
adds support for x86_64, ppc64 or other 64 bit architectures using /usr/lib64 for
their libraries. Having lib64 in front of lib is mandatory, as 64 bit systems are
likely to run with multi-lib, that means 32 bit libraries are parallel in use. If
you apply this patch, don't run autoconf > 2.6.3, as it will break, because whole
autotools stuff from upstream is unfortunately a bit broken.

--- unicornscan-0.4.7/configure.ac		2007-11-29 15:51:20.000000000 +0100
+++ unicornscan-0.4.7/configure.ac.lib64	2009-11-16 23:32:27.000000000 +0100
@@ -269,7 +269,7 @@
       AC_MSG_RESULT(no)
     fi
   else
-    for i in lib lib/mysql; do
+    for i in lib64 lib lib64/mysql lib/mysql; do
       str="$MYSQL_DIR/$i/libmysqlclient.*"
       for j in `echo $str`; do
         if test -r $j; then
@@ -331,7 +331,7 @@
       AC_MSG_RESULT(no)
     fi
   else
-    for i in lib lib/postgresql; do
+    for i in lib64 lib lib64/postgresql lib/postgresql; do
       str="$PGSQL_DIR/$i/libpq.*"
       for j in `echo $str`; do
         if test -r $j; then
--- unicornscan-0.4.7/configure			2007-12-18 16:26:12.000000000 +0100
+++ unicornscan-0.4.7/configure.lib64		2009-12-05 12:04:32.000000000 +0100
@@ -26275,7 +26275,7 @@
 echo "${ECHO_T}no" >&6; }
     fi
   else
-    for i in lib lib/mysql; do
+    for i in lib64 lib lib64/mysql lib/mysql; do
       str="$MYSQL_DIR/$i/libmysqlclient.*"
       for j in `echo $str`; do
         if test -r $j; then
@@ -26438,7 +26438,7 @@
 echo "${ECHO_T}no" >&6; }
     fi
   else
-    for i in lib lib/postgresql; do
+    for i in lib64 lib lib64/postgresql lib/postgresql; do
       str="$PGSQL_DIR/$i/libpq.*"
       for j in `echo $str`; do
         if test -r $j; then
